
LOG:  database system was shut down at 2016-01-23 16:20:31 PST
LOG:  MultiXact member wraparound protections are now enabled
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
FATAL:  database "Wayne" does not exist
FATAL:  database "pinpointdb" does not exist
LOG:  using stale statistics instead of current ones because stats collector is not responding
FATAL:  lock file "postmaster.pid" already exists
HINT:  Is another postmaster (PID 8499) running in data directory "/Users/Wayne/Desktop/pinpoint/pinpointdb"?
FATAL:  role "postgres" does not exist
FATAL:  database "Wayne" does not exist
FATAL:  database "pinpoint" does not exist
FATAL:  database "pinpointd" does not exist
LOG:  database system was interrupted; last known up at 2016-01-25 15:44:11 PST
LOG:  database system was not properly shut down; automatic recovery in progress
LOG:  invalid record length at 0/179CA18
LOG:  redo is not required
LOG:  MultiXact member wraparound protections are now enabled
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
ERROR:  syntax error at or near "/" at character 1
STATEMENT:  /dt
  ;
LOG:  database system was interrupted; last known up at 2016-01-25 17:09:57 PST
LOG:  database system was not properly shut down; automatic recovery in progress
LOG:  invalid record length at 0/17DC728
LOG:  redo is not required
LOG:  MultiXact member wraparound protections are now enabled
LOG:  autovacuum launcher started
LOG:  database system is ready to accept connections
ERROR:  column Users.username does not exist at character 43
STATEMENT:  SELECT "id" FROM "Users" AS "Users" WHERE "Users"."username" = 1 LIMIT 1;
ERROR:  column Users.username does not exist at character 43
STATEMENT:  SELECT "id" FROM "Users" AS "Users" WHERE "Users"."username" = 2 LIMIT 1;
ERROR:  relation "users" does not exist at character 15
STATEMENT:  select * from users;
ERROR:  relation "users" does not exist at character 15
STATEMENT:  select * from Users;
ERROR:  syntax error at or near "select" at character 47
STATEMENT:  ALTER SEQUENCE "Visits_id_seq" RESTART WITH 1
  select * from "Visits";
ERROR:  duplicate key value violates unique constraint "Users_pkey"
DETAIL:  Key (id)=(2) already exists.
STATEMENT:  INSERT INTO "Users" DEFAULT VALUES RETURNING *;
LOG:  database system was interrupted; last known up at 2016-01-27 12:00:44 PST
LOG:  database system was not properly shut down; automatic recovery in progress
LOG:  invalid record length at 0/19328F8
LOG:  redo is not required
LOG:  MultiXact member wraparound protections are now enabled
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
FATAL:  lock file "postmaster.pid" already exists
HINT:  Is another postmaster (PID 708) running in data directory "/Users/Wayne/Desktop/pinpoint/pinpointdb"?
FATAL:  database "pinpoint" does not exist
FATAL:  database "pinpointd" does not exist
ERROR:  schema "pinpointdb" does not exist
STATEMENT:  drop schema pinpointdb cascade;
ERROR:  relation "Users" does not exist at character 13
STATEMENT:  DELETE FROM "Users"
ERROR:  relation "Visits" does not exist at character 13
STATEMENT:  DELETE FROM "Visits"
ERROR:  duplicate key value violates unique constraint "Tags_name_key"
DETAIL:  Key (name)=(football) already exists.
STATEMENT:  INSERT INTO "Tags" ("id","name") VALUES (DEFAULT,'football'),(DEFAULT,'badminton');
FATAL:  lock file "postmaster.pid" already exists
HINT:  Is another postmaster (PID 708) running in data directory "/Users/Wayne/Desktop/pinpoint/pinpointdb"?
LOG:  database system was shut down at 2016-01-28 19:59:05 PST
LOG:  MultiXact member wraparound protections are now enabled
LOG:  database system is ready to accept connections
LOG:  autovacuum launcher started
LOG:  incomplete startup packet
LOG:  incomplete startup packet
LOG:  incomplete startup packet
LOG:  could not bind IPv6 socket: Address already in use
HINT:  Is another postmaster already running on port 5432? If not, wait a few seconds and retry.
LOG:  could not bind IPv4 socket: Address already in use
HINT:  Is another postmaster already running on port 5432? If not, wait a few seconds and retry.
LOG:  could not bind IPv6 socket: Address already in use
HINT:  Is another postmaster already running on port 5432? If not, wait a few seconds and retry.
WARNING:  could not create listen socket for "localhost"
FATAL:  could not create any TCP/IP sockets

FATAL:  lock file "postmaster.pid" already exists
HINT:  Is another postmaster (PID 708) running in data directory "/Users/Wayne/Desktop/pinpoint/pinpointdb"?
